lib/git-manager.js

```diff
@@ -61,11 +80,9 @@ module.exports = {
     * Returns a promise which resolves to a github issue
     */
    getIssue: function(repo, number) {
-      const searchParams = params({ number }, repo);
+      const searchParams = params({ issue_number: number }, repo);
       debug("Getting issue %s in repo %s", number, repo);
-      return rateLimit(github.issues.get)(searchParams)
-                          .then(res => res.data)
-                          .then(addRepo(searchParams));
+      return github.paginate(githubRest.issues.get, searchParams).then(addRepo(searchParams));
```

Comment:
I don't know that this makes sense to be paginated; I think `githubRest.issues.get` is expected to return a single issue.

Reply:
https://github.com/iFixit/pulldasher/pull/211/commits/b69f41b5d1db5dbacb4309e93edd3467d9282cad